packages/core/src/i18n/get-localization.ts

```diff
@@ -31,20 +34,22 @@ import GetLocalizationProps from '../models/get-localization-props';
  * @returns {Promise<Customization>} A promise that resolves with the merged branding properties.
  */
 const getLocalization = async (props: GetLocalizationProps): Promise<TextObject> => {
-  const {componentCustomization, locale, providerCustomization, screen} = props;
+  const {componentTextOverrides, locale, providerTextOverrides, screen} = props;
 
   const module: TextObject = await import(`./screens/${screen}/${locale}.ts`);
 
   let textFromConsoleBranding: BrandingPreferenceTextAPIResponse;
 
+  const configData: AuthClientConfig<UIAuthConfig> = await AuthClient.getInstance().getDataLayer().getConfigData();
+
   try {
-    if ((await AuthClient.getInstance().getDataLayer().getConfigData()).enableConsoleTextBranding ?? true) {
-      textFromConsoleBranding = await getBrandingPreferenceText(
+    if (configData.enableConsoleTextBranding ?? true) {
+      textFromConsoleBranding = await getBrandingPreferenceText({
         locale,
-        providerCustomization.name,
+        name: configData.name ?? 'carbon.super',
         screen,
-        providerCustomization.type,
-      );
+        type: configData.type ?? BrandingPreferenceTypes.Org,
```

Comment:
Let's check and nullable checks if required.

Reply:
Wil address this in the next PR.

Reply:
The config cannot be nullable. Application cannot reach this far, if this is nullable. I think it is expected to break, in case this becomes nullable.